crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -113,55 +119,97 @@ impl CompressedDataManager {
         StorageEngine::write_batch_to_apache_parquet_file(record_batch, file_path.as_path(), None)
     }
 
+    /// Read and process messages received from the [`super::UncompressedDataManager`] to either
+    /// ingest compressed data, flush buffers, or stop.
```

Comment:
Minor but maybe change `ingest` to `insert` since we have consistently used `ingest` for uncompressed data coming from outside ModelarDB.

Reply:
Fixed.